randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1110,8 +1110,36 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
       Trivial
   > Pickup (Energy Tank)
       All of the following:
-          Flash Shift and Morph Ball
-          Grapple Beam or Spider Magnet or Walljump (Beginner)
+          Morph Ball
+          Any of the following:
+              All of the following:
+                  Use Spin Boost
+                  Movement (Intermediate) or Slide Jump (Beginner)
+              All of the following:
+                  Speed Booster and Disabled Door Lock Randomizer
+                  Spider Magnet or Movement (Beginner)
+              All of the following:
+                  Flash Shift
+                  Any of the following:
+                      Spider Magnet
+                      All of the following:
+                          Movement (Beginner)
```

Comment:
not sure what the movement here is for. I understand for Speed Booster, but i personally dont feel it's needed for flash

Reply:
I was only able to get the walljump or spin boost off correctly if I did the flash from a height that has precision to it.